packages/web/src/public/locales/en/common.json

```diff
@@ -42,6 +43,7 @@
     "login": "Login",
     "manage_language": "Manage {{languageName}}",
     "new_language": "Create Language",
+    "reset_password": "Reset Password",
```

Comment:
I couldn't find usages for this one? If they exist, replace with `users:reset_password`

Reply:
Done.